README.md

```diff
@@ -153,7 +153,7 @@ developers, especially new contributors looking for something to work on, we do:
 - Code verification with [`Kani`](https://github.com/model-checking/kani)
 
 There are always more tests to write and more bugs to find, contributions to our testing efforts
-extremely welcomed. Please consider testing code a first class citizen, we definitely do take PRs
+are extremely welcomed. Please consider testing code a first class citizen, we definitely do take PRs
```

Comment:
In 9333f0a578f3314c6ba73036aa8be31f15f5b484:

This was a run-on before you changed it, and it's still a run-on. It should be split into two sentences, and the second one needs to be edited somehow, maybe to just "PRs are extremely welcomed."

All the other comma additions I could go either way on. I think they're all fine as-is but I don't mind them.

Reply:
The following sentence is also a run-on.

Reply:
Ok, the new text is much better.

Can you update your commit message to be more descriptive? Then I can ACK this.